frontend/ui/widget/bookstatuswidget.lua

```diff
@@ -42,59 +43,27 @@ local stats_book = {}
 },]]
 local BookStatusWidget = FocusManager:extend{
     padding = Size.padding.fullscreen,
-    settings = nil,
-    thumbnail = nil,
-    props = nil,
     star = nil, -- Button
     summary = nil, -- hash
 }
 
 function BookStatusWidget:init()
-    self.updated = false
+    self.updated = nil
     self.layout = {}
-    -- What a blank, full summary table should look like
-    local new_summary = {
-        rating = nil,
-        note = nil,
-        status = "",
-        modified = "",
-    }
-    if self.settings then
-        local summary = self.settings:readSetting("summary")
-        -- Check if the summary table we get is a full one, or a minimal one from CoverMenu...
```

Comment:
Is that no longer true?

I don't entirely recall the details, but I do remember that what we got from CoverMenu (IIRC, for books that were never opened by anythi_ng else than the thumbnailer), the table was very bare-bones.

Reply:
It's an old piece of code.
Currently we build/fill the summary immediately on book opening
https://github.com/koreader/koreader/blob/a51bd7a7fd1376ca972171994a6f6a7eec7a6464/frontend/apps/reader/readerui.lua#L488-L492